apps/system/js/fx_accounts_client.js

```diff
@@ -82,6 +82,12 @@ var FxAccountsClient = function FxAccountsClient() {
     }, successCb, errorCb);
   };
 
+  var getKeys = function getKeys(successCb, errorCb) {
```

Comment:
Well after looking again it appears this entire file is callback based, so probably fine for now. Would be nice in the future though.